modules/rostests/apitests/crt/crtdata.c

```diff
@@ -0,0 +1,640 @@
+    ok_ptr(*__p__pctype(), __pctype_func());
+#endif
+}
+
+void Test__wctype()
+{
+    ok_int(_wctype[0], 0);
+    ok_int(_wctype[1], _CONTROL);
+
+    #undef _pwctype
+    _CRTIMP extern const unsigned short* _pwctype;
+    ok_ptr(_pwctype, &_wctype[1]);
+
+    ok(__pwctype_func() != _wctype, "__pwctype_func() == _wctype\n");
+    ok_int(__pctype_func()[0], _CONTROL);
+    ok_int(__pctype_func()[1], _CONTROL);
+#ifdef _M_IX86
+    _CRTIMP const unsigned short** __cdecl __p__pwctype(void);
+    ok_ptr(*__p__pwctype(), __pwctype_func());
+#endif
+}
+
+void Test__daylight()
+{
+    void* p = &_daylight;
+    test_is_local_symbol(&_daylight, FALSE);
+
+    #undef _daylight
+    _CRTIMP extern int _daylight;
+    ok_ptr(&_daylight, p);
+
+#ifdef _M_IX86
+    _CRTIMP void* __p__daylight(void);
+    ok_ptr(__p__daylight(), &_daylight);
+#endif
+#if (WINVER >= 0x600)
+    _CRTIMP int* __cdecl __daylight(void);
+    ok_ptr(&__daylight, &_daylight);
+#endif
+}
+
+void Test__dstbias()
+{
+    void* p = &_dstbias;
+    test_is_local_symbol(&_dstbias, FALSE);
+
+    #undef _dstbias
+    _CRTIMP extern long _dstbias;
+    ok_ptr(&_dstbias, p);
+#ifdef _M_IX86
+    _CRTIMP long* __cdecl __p__dstbias(void);
+    ok_ptr(__p__dstbias(), &_dstbias);
+#endif
+#if (WINVER >= 0x600)
+    _CRTIMP long* __cdecl __dstbias(void);
+    ok_ptr(&__dstbias, &_dstbias);
+#endif
+}
+
+void Test__environ()
+{
+    void* p = &_environ;
+    ok(_environ != NULL, "_environ == NULL\n");
+
+    #undef _environ
+    _CRTIMP extern char** _environ;
+    ok_ptr(&_environ, p);
+#ifdef _M_IX86
+    ok_ptr(__p__environ(), &_environ);
+#endif
+}
+
+void Test__fileinfo()
+{
+    _CRTIMP extern int _fileinfo;
+    ok_int(_fileinfo, -1);
+
+#ifdef _M_IX86
+    _CRTIMP int* __p__fileinfo();
+    ok_ptr(__p__fileinfo(), &_fileinfo);
+#endif
+}
+
+void Test__fmode()
+{
+    void* p = &_fmode;
+    test_is_local_symbol(&_fmode, FALSE);
+    ok_int(_fmode, 0);
+
+    #undef _fmode
+    _CRTIMP extern int _fmode;
+    ok_ptr(&_fmode, p);
+
+#ifdef _M_IX86
+    _CRTIMP int* __cdecl __p__fmode();
+    ok_ptr(__p__fmode(), p);
+#endif
+
+#if (_WIN32_WINNT >= 0x600)
+    _fmode = 1234;
+    _CRTIMP errno_t __cdecl _get_fmode(_Out_ int* _PMode);
+    int mode;
+    ok_int(_get_fmode(&mode), 0);
+    ok_int(mode, _fmode);
+    _fmode = 0;
+#endif
+}
+
+void Test__iob()
+{
+    void* p = &_iob;
+    test_is_local_symbol(&_iob, FALSE);
+    ok_ptr(&_iob[0], stdin);
+    ok_ptr(&_iob[1], stdout);
+    ok_ptr(&_iob[2], stderr);
+
+    #undef _iob
+    _CRTIMP extern FILE _iob[];
+    ok_ptr(&_iob, p);
+
+    ok_ptr(__iob_func(), &_iob);
+
+#ifdef _M_IX86
+    _CRTIMP int* __cdecl __p__iob();
+    ok_ptr(__p__iob(), p);
+#endif
+}
+
+void Test__mbcasemap()
+{
+    void* p = &_mbcasemap;
+    ok_int(_mbcasemap[0], 0);
+
+    #undef _mbcasemap
+    ok_ptr(_mbcasemap, p);
+
+#ifdef _M_IX86
+    _CRTIMP unsigned char* __cdecl __p__mbcasemap();
+    ok_ptr(__p__mbcasemap(), &_mbcasemap);
```

Comment:
https://reactos.org/testman/detail.php?id=54128779&prev=0

Why & how is this marked todo ?!

Reply:
That was by accident. Fixed.